### DISCUSSION TITLE
What does the exact "TeeterDOF" mean in a 2-bladed wind turbine?

### DISCUSSION THREAD

**Discussion by Melancho7ee** [OpenFAST/openfast#1744] [General]
Since I am not an English native speaker, I failed to capture the real meaning of the teeter motion of a 2-bladed wind turbine. Could anyone help me to understand this definition? Many thanks if  pictures or GIFs provided.

**Comment by jjonkman** [comment]
Dear @Melancho7ee,

Teeter is unique feature of some two-bladed rotors where a rotational joint (teeter pin) is added between the shaft and hub to prevent non-torque bending loads from transferring to the shaft from the rotor.  See Figures 14 and 15 and accompanying text from the old FAST v6 User's Guide for more information: https://openfast.readthedocs.io/en/main/_downloads/d8bd014121d6505cb25cf49bee5eaa80/Old_FAST6_UsersGuide.pdf.

Best regards,
